script.js.js

```diff
@@ -0,0 +1,162 @@
+//   span.addEventListener('keydown', (event) => {
+//     if (event.key === 'Enter') {
+//       event.preventDefault();
+//       span.blur();
+//     }
+//   });
+//   // When the span element loses focus, check if the new description is empty, and if not, update the task description
+//   span.addEventListener('blur', () => {
+//     const newDescription = span.textContent.trim();
+//     if (newDescription === '') {
+//       deleteTask(index);
+//     } else {
+//       editTask(index, newDescription);
+//     }
+//   });
+// }
+// // This function is called when a user clicks on a task or its associated icons
+// function toggleItem(e) {
+//   // Find the closest li element to the clicked element
+//   const li = e.target.closest('li');
+//   if (li) {
+//     // Toggle the "selected" class of the li element
+//     li.classList.toggle('selected');
+//     // Find the ellipsis and trash icons within the li element
+//     const ellipsisIcon = li.querySelector('.fa-ellipsis-v');
+//     const trashIcon = li.querySelector('.fa-trash');
+//     if (li.classList.contains('selected')) {
+//       // If the li element has the "selected" class, hide the ellipsis icon and show the trash icon
+//       ellipsisIcon.style.display = 'none';
+//       trashIcon.style.display = 'block';
+//       trashIcon.style.cursor = 'pointer';
+//     } else {
+//       // If the li element does not have the "selected" class, show the ellipsis icon and hide the trash icon
+//       ellipsisIcon.style.display = 'block';
+//       trashIcon.style.display = 'none';
+//     }
+//   }
+//   // If the clicked element is an input element, update the completed status of the task
+//   if (e.target.tagName === 'INPUT') {
+//     const { index } = e.target.dataset;
+//     tasks[index].completed = e.target.checked;
+//     saveTasks();
+//     renderTasks();
+//   } else if (e.target.classList.contains('fa-ellipsis-v')) {
+//     // If the clicked element is the ellipsis icon, move the task up one position in the list
+//     const item = e.target.parentNode;
+//     const prev = item.previousElementSibling;
+//     todoList.insertBefore(item, prev);
+//     updateTasksIndexes();
+//   } else if (e.target.classList.contains('fa-trash')) {
+//     // If the clicked element is the trash icon, delete the task
+//     const item = e.target.parentNode;
+//     const { index } = item.querySelector('input').dataset;
+//     deleteTask(index);
+//   }
+// }
+// // Add event listeners to various elements on the page
+// form.addEventListener('submit', addItem);
+// todoList.addEventListener('click', toggleItem);
+// todoList.addEventListener('dblclick', editTaskDescription);
+// archiveBtn.addEventListener('click', clearCompleted);
+// refreshIcon.addEventListener('click', refreshPage);
+// downArrowIcon.addEventListener('click', addItem);
+// // Render the initial tasks on the page when it loads
+// renderTasks();
```

Comment:
Hi @Yordinia,

- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain.